src/components/InputTodo.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import React, { Component } from 'react';
```

Comment:
it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules.